### PR TITLE
Create search component

### PR DESCRIPTION
**Pull request by RicardoBachmann**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -50,9 +52,20 @@ export default function App() {
...
 
+  function search(participantData) {
+    return participantData.filter(participant => {
+      const address = participant.properties.address.toLowerCase();
+      const name = participant.properties.business_name?.toLowerCase();
+
+      return address.includes(query) || (name && name.includes(query));
+    });
+  }
+  console.log(query);
```
Comment (stefankorder) [inline src/App.js:63]: Please delete this console log. In your final PR there should be no console logs.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work! Just one console log.

File: src/App.js (hunk 1 above)
Comment (RicardoBachmann) [reply src/App.js:63]: Ah yes! i will delete this console log..
But there is something strange around this filter function.. if i type capital letters in my input field there are no results. Lower case letters are working..

Comment (RicardoBachmann) [reply src/App.js:63]: Is fixed

**Pull request merged**